rmgpy/rmg/input.py

```diff
@@ -192,6 +194,11 @@ def species(label, structure, reactive=True):
     rmg.initial_species.append(spec)
     species_dict[label] = spec
 
+def forbidden(label, structure):
+
+    if '+' in label:
```

Comment:
This probably isn't wrong, but I don't understand why we are specifically excluding "+" . Does it cause concatenation somewhere else if we don't have this check?